packages/tabs/src/tab-nav.vue

```diff
@@ -113,11 +113,6 @@
         }
       }
     },
-
-    updated() {
-      this.update();
```

Comment:
sorry, there is someting wrong with my pr,  what  I'm curious about is that why the update method wrapped in updated function, I mean it could be invoked directly in mounted hooks